src/lib/components/ui/ArticleList.svelte

```
@@ -1,129 +1,131 @@
+		} finally {
+			loading = false;
+		}
+	}
+
+	$effect(() => {
+		visibleArticles = selectedCategory ? Number.MAX_SAFE_INTEGER : articlesPerPage;
+	});
+
+	$effect(() => {
+		if (filteredArticles.length > previousVisibleCount) {
+			newArticleRef = document.getElementById(`article-${previousVisibleCount}`);
```

ESLint

[error] 'newArticleRef' is not defined.

(no-undef)